src/WarcraftLegacies.Source/Quests/Scourge/QuestEnemyEncroachment.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MacroTools.FactionSystem;
+using MacroTools.ObjectiveSystem.Objectives.UnitBased;
+using MacroTools.QuestSystem;
+
+namespace WarcraftLegacies.Source.Quests.Scourge
+{
+  public sealed class QuestEnemyEncroachment : QuestData
+  {
+    
+    /// <inheritdoc />
+    public QuestEnemyEncroachment(Faction faction) : base("Enemy Encroachment", "Hostile forces have stepped foot onto Northrend! The Frozen Throne will have to employ new powers to defend against these interlopers", @"ReplaceableTextures\CommandButtons\BTNAnimateDead.blp")
+    {
+      AddObjective(new ObjectiveAnyEnemyUnitInRects(new[]
+      {
+        Regions.Storm_Peaks,
+        Regions.Central_Northrend,
+        Regions.The_Basin,
+        Regions.Ice_Crown,
+        Regions.Fjord,
+        Regions.Eastern_Northrend,
+        Regions.Far_Eastern_Northrend,
+        Regions.Coldarra,
+        Regions.Borean_Tundra,
+        Regions.IcecrownShipyard
+      },faction, "Northrend", false));
+      ResearchId = Constants.UPGRADE_R08L_DEACTIVATED;
+    }
+    
+    protected override string RewardFlavour => "Frozen Throne can now use Frost Nova and Animate Dead spells to defend Northrend from invaders.";
```

Comment:
Reward flavour should be flavour only; it can pertain to the mechanics of the quest but cannot explicitly call out mechanical terms.

Reply:
```suggestion
    protected override string RewardFlavour => "For the first time in years, outlanders have set foot on the shores of Northrend. Soon they will feel the biting chill of death.";
```